autoPyTorch/evaluation/abstract_evaluator.py

```diff
@@ -247,13 +247,90 @@ def send_warnings_to_log(message, category, filename, lineno,
 
 
 class AbstractEvaluator(object):
+    """
+    This method defines the interface that pipeline evaluators should follow, when
+    interacting with SMAC through ExecuteTaFuncWithQueue.
+
+    An evaluator is an object that:
+        + constructs a pipeline (be it a classification or regression estimator) for a given
+          pipeline_config and run settings (budget, seed)
+        + Fits and train this pipeline (TrainEvaluator) or test a given
+          configuration (TestEvaluator)
+
+    The provided configuration determines the type of pipeline created:
+        int: A dummy classifier/dummy regressor is created. This estimator serves
+             as a baseline model to ignore all models that perform worst than this
+             dummy estimator. Also, in the worst case scenario, this is the final
+             estimator created (for instance, in case not enough memory was allocated
+             and all configurations crashed).
+        str: A traditional pipeline is created, as the configuration will contain a
```

Comment:
A pipeline with traditional classifiers like random forest, SVM, etc is created, as the configuration ...

in the end maybe we can add something like for more information on the available classifiers, check out autoPyTorch/pipeline/components/setup/traditional_ml/classifier_models/__init__.py